index.js

```diff
@@ -0,0 +1,95 @@
+var express = require('express')
+var bodyParser = require('body-parser')
+var models = require('./models')
+
+var app = express()
+
+app.get('/movies', async (request, response) => {
+    const matchingmovies = await models.movies.findAll({
+        include: [{ model: models.directors }, { model: models.genres }]
+    })
+    return response.send(matchingmovies)
+})
+
+app.get('/movies/:identifier', async (request, response) => {
+    const movies = await models.movies.findAll({
+        include: [{
+            all: true
```

Comment:
I find all gets messy because it includes the linking table records. Better to call out directors and genres like you did in the other movie get